### PR TITLE
WIP waiting for community feedback: MK3 PFW-1445 shorten translations

### PR DESCRIPTION
**Pull request by 3d-gussner**
Shorten messages

### CODE REVIEW

**Comment by 3d-gussner** [comment]
Dear community translators
@vintagepc @stelgenhof @leptun @Hauzman @AttilaSVK @Prime1910 @ingbrzy @Painkiller56 @OS-kar @trondkla
I have prepared the translations of new messages and would like to ask you t review the po file changes. Please comment here if these are OK or which cahgnes have to be made. Keep in mind that some had to be shorten due to `c=xx` limits.

Blow you find the output of all messages as shown on LCD screen.

Thanks  

[20230310_cs.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940408/20230310_cs.zip)
[20230310_de.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940410/20230310_de.zip)
[20230310_es.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940411/20230310_es.zip)
[20230310_fr.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940412/20230310_fr.zip)
[20230310_hr.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940413/20230310_hr.zip)
[20230310_hu.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940414/20230310_hu.zip)
[20230310_it.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940415/20230310_it.zip)
[20230310_nl.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940416/20230310_nl.zip)
[20230310_no.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940417/20230310_no.zip)
[20230310_pl.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940418/20230310_pl.zip)
[20230310_ro.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940419/20230310_ro.zip)
[20230310_sk.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940420/20230310_sk.zip)
[20230310_sv.zip](https://github.com/prusa3d/Prusa-Firmware/files/10940421/20230310_sv.zip)

**Comment by 3d-gussner** [comment]
@ErwanAliasr1 These are the new French po files please have a look and if you like comment here what you would like to be changed. As this is an official supported language your suggestions will be reviewed by internal translators.

File: Firmware/mmu2.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ struct Version {
 };
 
 /// Top-level interface between Logic and Marlin.
-/// Intentionally named MMU2 to be (almost) a drop-in replacement for the previous implementation.
+/// Intentionally named MMU to be (almost) a drop-in replacement for the previous implementation.
```
Comment (DRracer) [inline Firmware/mmu2.h]: tbh. I wouldn't change MMU2->MMU in this case :wink: - it is a comment about the MMU2 class

File: Firmware/mmu2/errors_list.h [hunk 2]
```diff
@@ -255,17 +255,17 @@ static const char MSG_DESC_TMC[] PROGMEM_I1 = ISTR("More details online."); ////
...
 //static const char MSG_DESC_IDLER_TMC_DRIVER_SHORTED[] PROGMEM_I1 = ISTR("Short circuit on the Idler TMC driver. Check the wiring and connectors. If the issue persists contact support.");
 //static const char MSG_DESC_MCU_UNDERVOLTAGE_VCC[] PROGMEM_I1 = ISTR("MMU MCU detected a 5V undervoltage. There might be an issue with the electronics. Check the wiring and connectors"); ////MSG_DESC_MCU_UNDERVOLTAGE_VCC c=20 r=8
-static const char MSG_DESC_MMU_NOT_RESPONDING[] PROGMEM_I1 = ISTR("MMU unit not responding. Check the wiring and connectors. If the issue persists, contact support."); ////MSG_DESC_MMU_NOT_RESPONDING c=20 r=8
-static const char MSG_DESC_COMMUNICATION_ERROR[] PROGMEM_I1 = ISTR("MMU unit not responding correctly. Check the wiring and connectors. If the issue persists, contact support."); ////MSG_DESC_COMMUNICATION_ERROR c=20 r=9
+static const char MSG_DESC_MMU_NOT_RESPONDING[] PROGMEM_I1 = ISTR("MMU not responding. Check the wiring and connectors. If the issue persists, contact support."); ////MSG_DESC_MMU_NOT_RESPONDING c=20 r=8
```
Comment (DRracer) [inline Firmware/mmu2/errors_list.h]: Have you checked, that the same changes landed into the Prusa-Error-Codes repo?
`errors_list.h` should have been build-time generated, which is currently impossible due to all the hand-crafted changes we had to do to squeeze it into the small AVR CPU. But that doesn't mean that Prusa-Error-Codes and this file should differ.

Comment (3d-gussner) [reply Firmware/mmu2/errors_list.h]: https://github.com/prusa3d/Prusa-Error-Codes/pull/82

**Comment by Painkiller56** [comment]
Swedish translation corrections;

 

Line 880: Cosmetic space between "." and "S"

   3 |filamment. Se till  |

 

Line 1081: Cosmetic space

Line 1101: Same

   7 |sensorn fungerar.   |

 

Line 1191-1192

   2 |munstycke, se       |

   3 |prusa.io/nozzle-mk3s|

Line 1193: delete

 

-------------------

Following three; Search for "G-codeen" and replace automatic with "G-coden".

 

Line 2683: one "e" too much in codeen

   3 |G-coden. Fortsätta? |

 

Line 2698: one "e" too much in codeen

   3 |fran G-coden.       |

 

Line 2798: one "e" too much in codeen

   5 |Kontrollera G-coden |

--------------------

 

Line 3993: 

   |FILAMENT UTMATAT    |

 

Line 4064-4065:

   1 |Ta bort det         |

   2 |utmatade filamentet |

 

Line 4092-4093:

   4 |filament. Se till   |

   5 |att inget filament  |

 

 

Sincerely,

 

Bernt

 

 

Från: 3d-gussner ***@***.***> 
Skickat: den 10 mars 2023 10:18
Till: prusa3d/Prusa-Firmware ***@***.***>
Kopia: Bernt Ahblbäck ***@***.***>; Mention ***@***.***>
Ämne: Re: [prusa3d/Prusa-Firmware] :construction: WIP: MK3 PFW-1445 shorten translations (PR #4073)

 

Dear community translators
@vintagepc <https://github.com/vintagepc>  @stelgenhof <https://github.com/stelgenhof>  @leptun <https://github.com/leptun>  @Hauzman <https://github.com/Hauzman>  @AttilaSVK <https://github.com/AttilaSVK>  @Prime1910 <https://github.com/Prime1910>  @ingbrzy <https://github.com/ingbrzy>  @Painkiller56 <https://github.com/Painkiller56>  @OS-kar <https://github.com/OS-kar>  @trondkla <https://github.com/trondkla> 
I have prepared the translations of new messages and would like to ask you t review the po file changes. Please comment here if these are OK or which cahgnes have to be made. Keep in mind that some had to be shorten due to c=xx limits.

Blow you find the output of all messages as shown on LCD screen.

Thanks

20230310_cs.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940408/20230310_cs.zip> 
20230310_de.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940410/20230310_de.zip> 
20230310_es.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940411/20230310_es.zip> 
20230310_fr.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940412/20230310_fr.zip> 
20230310_hr.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940413/20230310_hr.zip> 
20230310_hu.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940414/20230310_hu.zip> 
20230310_it.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940415/20230310_it.zip> 
20230310_nl.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940416/20230310_nl.zip> 
20230310_no.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940417/20230310_no.zip> 
20230310_pl.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940418/20230310_pl.zip> 
20230310_ro.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940419/20230310_ro.zip> 
20230310_sk.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940420/20230310_sk.zip> 
20230310_sv.zip <https://github.com/prusa3d/Prusa-Firmware/files/10940421/20230310_sv.zip>

**Comment by Hauzman** [comment]
@3d-gussner Ro seam's to be OK

**Comment by 3d-gussner** [comment]
@Painkiller56 Thanks for the review, good catch. See https://github.com/prusa3d/Prusa-Firmware/pull/4073/commits/47f0d281c2bfc1d9659e9bb7b30789e643c2014d and https://github.com/prusa3d/Prusa-Firmware/pull/4073/commits/ffdd6c2ead179466f7c45e66a680ad67d82b1912

**Comment by ingbrzy** [comment]
@3d-gussner fixed SK translation in attachment...

[Firmware_sk.zip](https://github.com/prusa3d/Prusa-Firmware/files/10947782/Firmware_sk.zip)

**Comment by 3d-gussner** [comment]
@AttilaSVK @Prime1910 @OS-kar @trondkla Please review these translations.

File: lang/po/Firmware_nl.po [hunk 3]
```diff
@@ -915,48 +915,49 @@ msgid "Homing"
 msgstr "Startpositie"
 
 #. MSG_NOZZLE_CNG_CHANGED c=20 r=6
-#: ../../Firmware/messages.cpp:174 ../../Firmware/ultralcd.cpp:1036
+#: ../../Firmware/messages.cpp:183 ../../Firmware/ultralcd.cpp:1000
 msgid "Hotend at 280C! Nozzle changed and tightened to specs?"
-msgstr "Hotend op 280C! Mondstuk vervangen en aangedraaid volgens specificaties?"
+msgstr ""
+"Hotend op 280C! Mondstuk vervangen en aangedraaid volgens specificaties?"
```
Comment (vintagepc) [inline lang/po/Firmware_nl.po]: Inconsistency, nozzle = "mondstuk" here and "tuit" later on

**Review by vintagepc — APPROVED** [review]
all the other changed items in the diff look okay :+1:

**Review by ingbrzy — APPROVED** [review]

**Review by shatter136 — APPROVED** [review]
thumbsup

**Comment by ErwanAliasr1** [comment]
I pushed https://github.com/ErwanAliasr1/Prusa-Firmware/commit/64d5842f3a314fba9a21a08e96c112a785089a5e for the French translation updates

**Comment by 3d-gussner** [comment]
> I pushed [ErwanAliasr1@64d5842](https://github.com/ErwanAliasr1/Prusa-Firmware/commit/64d5842f3a314fba9a21a08e96c112a785089a5e) for the French translation updates

@ErwanAliasr1 Thanks, that looks right to me. I will ask the internal translators to review your changes. Like that some messages got even shorter as French is using most space in translations.

**Comment by ErwanAliasr1** [comment]
@3d-gussner Updated the french po with your comments at https://github.com/ErwanAliasr1/Prusa-Firmware/commit/6f436230283dd3cdea50e94a0570fd13743183c3
Thanks for the review.

**Comment by 3d-gussner** [comment]
Closing in favor of  https://github.com/prusa3d/Prusa-Firmware/pull/4082

**Pull request closed without merging**